mbtiles/scripts/cli.py

```diff
@@ -72,20 +73,25 @@ def validate_nodata(dst_nodata, src_nodata, meta_nodata):
 @click.option('--resampling', type=click.Choice(RESAMPLING_METHODS),
               default='nearest', show_default=True,
               help="Resampling method to use.")
+@click.option('--bands', type=int,
+              default=None, show_default=True,
+              help="Number of bands in the output dataset (4 for RGBA)")
+@click.option('--colormap',
+              default=None, show_default=True,
+              help='''Colormap for a 1-band PNG file as a JSON array (e.g. {"0":[1,2,3,4],"23":[9,4,3,2]} for RGBA values for pixel values 0 and 23)''')
 @click.version_option(version=mbtiles_version, message='%(version)s')
-@click.option('--rgba', default=False, is_flag=True, help="Select RGBA output. For PNG only.")
```

Comment:
@eliask I would prefer to keep labels for the PNG varieties. "rgb" for 3 channel PNG (the default now), "rgba" for 4 channel (including alpha) PNG, and for the new one: what would you think about "paletted"? That's the term used in PNG docs and is already well known.